go_modules/spec/fixtures/projects/vendor/go.mod

```
@@ -0,0 +1,7 @@
+module github.com/dependabot/vgotest
+
+go 1.12
+
+require github.com/pkg/errors v0.9.1
```

```suggestion
require github.com/pkg/errors v0.8.0
```

~Or maybe the `let(:dependency_previous_version) { "v0.8.0" }` lies? I think they should be consistent.~
Edit: more sure this should be `v0.8.0`